src/manifolds/InvertibleMatrices.jl

```diff
@@ -0,0 +1,116 @@
+@doc raw"""
+    InvertibleMatrices{𝔽,T} <: AbstractDecoratorManifold{𝔽}
+
+The [`AbstractManifold`](@extref `ManifoldsBase.AbstractManifold`)  ``\operatorname{Sym}(n)``
```

Comment:
Maybe we should find a different symbol for this manifold?

Reply:
Oh just missed to remove this, since I did not find a good symbol here, I for now went with no symbol.